### PR TITLE
Create CaseMailer #3200

### PR DESCRIPTION
**Pull request by StlMaris123**
In your PR did you:
- Create CaseMailer
- Solves #3200
  - [x] Include a description of the changes?
  - [x] Mention the issue the PR addresses?
  - [ ] Include screenshots of any changes to the UI?
  - [ ] Isolate any changes to gems (meaning that any new, updated, or removed gems and resulting code changes should be in their own PR)?
  - [x] Add and/or update specs for your code?

### CODE REVIEW

**Comment by trystant** [comment]
@StlMaris123 have you run `rubocop -a` on the files in this PR?

**Comment by StlMaris123** [comment]
Hi @trystant , yes I did...It is a refactoring issue on the case mailer

**Comment by rlgreen91** [comment]
Heyo, so I would like to hold this PR - someone else is working on renaming this mailer from `UserNotifier` to `UserMailer`, and I'd like to merge his PR first.

**Comment by rlgreen91** [comment]
@StlMaris123 can you resolve these merge conflicts?  Then I'll be able to merge this.

File: app/mailers/case_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
```
Comment (rlgreen91) [inline app/mailers/case_mailer.rb]: Please use keyword arguments.

**Comment by rlgreen91** [comment]
Hey @StlMaris123 are you going to be able to finish this soon?

**Comment by rlgreen91** [comment]
@StlMaris123 one last thing - can you take care of the CodeClimate failures?  You can ignore the ones about similar blocks of code, but fix the line too long and missing top-level documentation one.

**Pull request merged**